src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -80,7 +80,13 @@ export class NavbarComponent implements OnInit, OnDestroy {
     subscribeForGuidedTourAvailability(): void {
         // Check availability after first subscribe call since the router event been triggered already
         this.guidedTourService.getGuidedTourAvailabilityStream().subscribe(isAvailable => {
-            this.isTourAvailable = isAvailable;
+            // The guided tour is currently disabled for mobile devices and tablets
+            // TODO optimize guided tour layout for mobile devices and tablets
+            if (this.deviceService.isMobile() || this.deviceService.isTablet()) {
```

Comment:
For me, it feels like this is the wrong place to add this check.
I would suggest to move this to a `.map` in the [guided-tour.service.ts#L82].

[guided-tour.service.ts#L82]: https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/guided-tour/guided-tour.service.ts#L82

Reply:
Thank you for the suggestion, I moved the logic to the `GuidedTourService` now.